tutorials/workflow/csharp/combined-patterns/order-workflow.http

```diff
@@ -0,0 +1,26 @@
+@apphost=http://localhost:5260
+
+### Start the OrderWorkflow workflow
+# @name startWorkflowRequest
+@orderId={{$guid}}
+POST {{ apphost }}/start
+Content-Type: application/json
+
+{
+    "id": "{{orderId}}",
+    "orderItem" : {
+        "productId": "RBD001",
+        "productName": "Rubber Duck",
+        "quantity": 10,
+        "totalPrice": 15.00
+    },
+    "customerInfo" : {
+        "id" : "Customer1",
+        "country" : "The Netherlands"
+    }
+}
+
+### Get the workflow status
+@instanceId={{startWorkflowRequest.response.headers.Location}}
+@daprHost=http://localhost:3560
+GET {{ daprHost }}/v1.0/workflows/dapr/{{ instanceId }}
```

Comment:
Might make sense to add a restock inventory call so that the users dont have to restart the application in order to restock the ducks :)